analysisstore/client/commands.py

```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```

Comment:
You could make it smart and look for a colon to split host/port, but that may make it needlessly complex